SyncUps/SyncUps/SyncUpsList.swift

```diff
@@ -1,6 +1,7 @@
 import Dependencies
 import DependenciesMacros
```

Comment:
Can this be removed?

```suggestion
```

Reply:
yep, now that we aren't using `@DependencyEndpoint`.